cmd/kubeadm/app/cmd/phases/workflow/runner.go

```diff
@@ -330,9 +330,14 @@ func (e *Runner) BindToCommand(cmd *cobra.Command) {
 	e.prepareForExecution()
 
 	// adds the phases subcommand
+	const phaseCommandUse = "phase"
 	phaseCommand := &cobra.Command{
-		Use:   "phase",
+		Use:   phaseCommandUse,
 		Short: fmt.Sprintf("Use this command to invoke single phase of the %s workflow", cmd.Name()),
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return errors.Errorf("%s\n\nphase name is required", e.Help(phaseCommandUse))
```

Comment:
RunE here just returns and error that has the constructed Help() screen.
it means that one should be calling the parent phase command without a subcommand.